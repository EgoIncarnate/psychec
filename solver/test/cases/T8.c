typedef struct A { float x; int y; } A;
void f() {
    A a;
    T1 i = a.x;
    A* ap;
    T2 j = ap->y;
}
