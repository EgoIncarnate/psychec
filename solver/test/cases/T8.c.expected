typedef int T2;
typedef float T1;
